src/ape/api/accounts.py

```diff
@@ -93,10 +93,16 @@ def call(self, txn: TransactionAPI, send_everything: bool = False) -> ReceiptAPI
         txn = self.prepare_transaction(txn)
 
         if send_everything:
+            error_message = "Sender does not have enough to cover transaction value and gas: "
             if txn.max_fee is None:
                 raise TransactionError(message="Max fee must not be None.")
-
-            txn.value = self.balance - txn.max_fee
+            if txn.gas_limit:
```

Comment:
Should this be `if txn.gas_limit is not None` ?

Reply:
i updated to this above and it is passing tests.